universe/UniverseObject.cpp

```diff
@@ -16,7 +16,7 @@
 #include "../util/i18n.h"
 
 namespace ValueRef {
-    const std::string& MeterToName(MeterType meter);
+    const std::string& MeterToName(const MeterType& meter);
```

Comment:
`MeterType` is an enum so is better passed by value in cases like this

Reply:
Good point, changed!